Services/IEventsService.cs

```diff
@@ -18,7 +18,9 @@ public interface IEventPartService
         /// <see cref="EventPart"/> start date and time.
         /// </param>
         /// <param name="afterFilterDate">
-        /// When set to <see afterFilterDate="false"/> the content item <see cref="EventPart"/> start date and time has
+        /// When <paramref name="afterFilterDate"/> set to <see langword="false"/> the content item

```

Comment:
```suggestion
        /// When <paramref name="afterFilterDate"/> is set to <see langword="false"/> the content item
```

Also in the other places.